string topicName
string MD5Sum
string dataType
uint8[] byteArray